docs/dm/assets.md

```diff
@@ -87,62 +98,72 @@ This menu includes a plethora of options.
 
 ### Is a token
 
-Selecting this option classifies the asset as a token. 
-This will help you keep track of which assets are tokens and which are other assets.
+Selecting this option classifies the object as a token. 
+Tokens function fundamentally differently than other shapes.
+
+Tokens cannot pass through shapes with the 'blocks movement' option enabled. 
+Tokens are also effected by lighting.
+Check the [lighting and vision](/docs/dm/light-shadows/) document for more information on how lighting works in relation to tokens.
 
 ### Is invisible
 
-Selecting this option turns the asset invisible to your players.
-As the DM, you will still be able to see the asset.
+Selecting this option turns the shape invisible to your players.
+As the DM, you will still be able to see the shape.
 This can be useful for hiding future encounters or placing enemies in stealth. 
```

Comment:
```suggestion
This can be useful for having objects that are invisible but player-controlled, or invisible, but still affecting movement. 
```
Hiding future encounters or placing enemies in stealth is the thing the DM layer was designed for, imho. :)